src/index.ts

```diff
@@ -108,5 +108,5 @@ export async function typescriptOfSchema (db: Database|string,
     return processedResult.dest
 }
 
-export {Database, getDatabase} from './schema'
-export {Options, OptionValues}
+export { Database, getDatabase } from './schema'
+export { Options, OptionValues }
```

Comment:
Linter was made at me.

Note that we also have some test failures from manual changes to this code that were never fixed.